quests/control_repository.md

```diff
@@ -417,7 +417,7 @@ use to authenticate itself with Gitea.
 
 Begin by creating a new `ssh` subdirectory in `/etc/puppetlabs/puppetserver`:
 
-    mkdir /etc/puppetlabs/puppetserver/ssh
+    mkdir -p /etc/puppetlabs/puppetserver/ssh
```

Comment:
@rutson Does the command fail without the `-p` option? Can you provide some more details here?

Reply:
It's a soft error, but it made me pause and double check.
![image](https://user-images.githubusercontent.com/14196062/101943267-b4fc1880-3be2-11eb-8d1a-532bcfc62f53.png)